helix-term/src/commands.rs

```diff
@@ -1519,7 +1519,10 @@ mod cmd {
         _event: PromptEvent,
     ) -> anyhow::Result<()> {
         let path = args.get(0).context("wrong argument count")?;
-        let _ = cx.editor.open(path.into(), Action::Replace)?;
+        let _ = cx.editor.open(
+            helix_core::path::expand_tilde(&PathBuf::from(path)),
```

Comment:
No need to allocate:

```suggestion
            helix_core::path::expand_tilde(Path::from(path)),
```

Reply:
I'd also import the `expand_tilde` function on line 1521

Reply:
@archseer Changes completed as requested, but `expand_tilde` does expect a reference, so I'm passing `&Path`.

Reply:
`Path::new` returns a reference. It's a bit surprising but that's because it's equivalent to `&str`

Reply:
https://doc.rust-lang.org/std/path/struct.Path.html#method.new

Reply:
Ah, my confusion came from the requested change (`Path::from` I think should be `Path::new`), and the difference between `PathBuf::from` (which returns an owned thing) and `Path::new`, which returns that reference). It should be all set now :)